commands/clear_reports.js

```diff
@@ -0,0 +1,57 @@
+
+const Discord = require('discord.js');
+async function runCommand(server, rcon, msg, toClear, reason) {
+    if(!rcon.connected){
+        await msg.channel.send(`S${server} is not connected the bot.`)
+        return;
+    }
+    let res = await rcon.send(`/interface require("modules.control.reports").remove_all("${toClear}", "${msg.member.displayName}")`)
+    await msg.channel.send(`*Please make sure you have the correct username and a report was issued in #reports.* Server replies complete even if user does not exist. Server replied: ${res} By: ${msg.member.displayName} becasue: *${reason}*`);
+    console.log(`A user tried to Clear ${toClear}'s reports... \n    Server replied: ${res}    By: ${msg.member.displayName}/${msg.member.id} \n    Becasue: *${reason}*`);
+    let reportChan = msg.guild.channels.cache.get('368812365594230788'); // Reports channel is "368812365594230788" for exp // Reports Channel is "764881627893334047" for test server
+}
+
+
+module.exports = {
+    name: 'clear-reports',
+    aka: ['clearreports','cr','clear_reports','clear-report'],
+    description: 'Clear reports on ingame users (Admin/Mod only command)',
+    guildOnly: true,
+    args: true,
+    helpLevel: 'role.staff',
+    required_role: role.staff,
+    usage: ` <username> <reason>`,
+    async execute(msg, args, rcons, internal_error) {
+        const author = msg.author.username; //find author
+        const server = Math.floor(Number(args[0]));
+
+        let reason = args.slice(2).join(" ");
+        let toClear = args[1];
+
+        if (!server) { // Checks to see if the person specified a server number
+            msg.channel.send('Please pick a server first (**just the number 1-8**). \`<Server#> <username> <reason>\`');
```

Comment:
msg.channel.send is a promise so must either be awaited or caught with internal error

Reply:
I can't find any docs that say channel.send() is a primise unless constructed with await - I know I'm missing something but I've never once catched a message.channel.send() (does not mean I'm doing it right though).

Reply:
no becuase they normanly dont error but msg.channel.send returns an promise so if it were to some how fail it would need to caugth with .catch

Reply:
```suggestion
            msg.channel.send('Please pick a server first (**just the number 1-8**). \`<Server#> <username> <reason>\`')
            .catch((err) => { internal_error(err); return });
```
@tovernaar123 like this?